stats/s-bandwidth

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env ruby
+LKP_SRC = ENV['LKP_SRC'] || File.dirname(File.dirname(File.realpath($PROGRAM_NAME)))
+
+require "#{LKP_SRC}/lib/statistics"
+
+workload = 0
+data = false
+pattern = /^\S*\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+(\d*\.?\d+)\s+/
```

Comment:
help add some comments to show the raw output that this pattern is to match e.g. 
# 02/15/21 17:38:55 Device tps MB_read/s MB_wrtn/s MB_dscd/s MB_read MB_wrtn MB_dscd nvme0n1 3366.00 136.23 0.03 0.00 3408 0 0

Reply:
Added a comment explaining in more detail the pattern to match